test/torchaudio_unittest/io/playback_test.py

```diff
@@ -0,0 +1,23 @@
+import torch
+import torchaudio
+from torchaudio_unittest.common_utils import (
+    skipIfNoDevice,
+    get_asset_path,
+    TorchaudioTestCase,
+    )
+from torchaudio.io import StreamWriter, play_audio
+from unittest.mock import patch
+
+
+@skipIfNoDevice
+class PlaybackInterfaceTest(TorchaudioTestCase):
+    @patch.object(StreamWriter, "write_audio_chunk")
+    def test_playaudio(self, writeaudio_mock):
+        """Test playaudio function"""
+        file = get_asset_path("steam-train-whistle-daniel_simon.mp3")
+        waveform, sample_rate = torchaudio.load(file)
```

Comment:
In the unit test, we want to limit the use of library function to what we are testing. (Otherwise, when this test fails, we cannot know immediately which function has the issue)

Can you use [`get_sinusoid`](https://github.com/pytorch/audio/blob/main/test/torchaudio_unittest/common_utils/data_utils.py#LL82C7-L82C7)  helper function to generate a test data?